doc/website/getting-started/examples/waitset_in_c.md

```diff
@@ -0,0 +1,5 @@
+---
+title: How to use the WaitSet in C
```

Comment:
Getting notified by events like the receiving of data (with the WaitSet) by using the C API.

Reply:
Waiting for data events in C

The waitset is the "how" again and the title covers the "what".

Or does this include user trigger events? I think this warrants its own section. The split could be similar in C++, but if we do not want the split just 

Using the WaitSet in C

would also be fine I guess.

Reply:
Probably the user triggers should be moved to it's own example, but not for the Almond release.